### DISCUSSION TITLE
Files created on WSL that end with "." can not be found when using CMD tools like dir, del.

### DISCUSSION THREAD

**Discussion by opedroso** [microsoft/WSL#8371] [General]
# Steps to Reproduce

## on Windows CMD.exe

cd c:\temp
echo hi > "hi.win."

## on WSL bash

cd /mnt/c/temp
echo hi > "hi.wsl."
opedroso@PAE-NB-9sE44Z6l:/mnt/c/temp$ ls -lsa hi.w*
0 -rwxrwxrwx 1 opedroso opedroso 5 May  5 07:20 hi.win
0 -rwxrwxrwx 1 opedroso opedroso 3 May  5 07:21 hi.wsl.

## On Windows CMD.exe

C:\temp>dir hi.*
 Volume in drive C is OS
 Volume Serial Number is E8E0-5FA9

 Directory of C:\temp

05.05.2022  07:20                 5 hi.win
05.05.2022  07:21                 3 hi.wsl.

C:\temp>del "hi.wsl."
Could Not Find C:\temp\hi.wsl.

C:\temp>dir hi*.*
 Volume in drive C is OS
 Volume Serial Number is E8E0-5FA9

 Directory of C:\temp

05.05.2022  07:20                 5 hi.win
05.05.2022  07:21                 3 hi.wsl.

## File ending in "." can be deleted from WSL bash

opedroso@PAE-NB-9sE44Z6l:/mnt/c/temp$ rm -v hi.w*
removed 'hi.win'
removed 'hi.wsl.'

**Comment by ghost** [comment]
I'm not sure what aspects of this have to do with WSL? If filing bugs in the future please file them using the bug template, if we were to try to reproduce something there's not much info to go off of here.
